### PR TITLE
Changed dataset from housing data to breast cancer

### PR DESCRIPTION
**Pull request by purujitb**

### CODE REVIEW

File: apphub/tabular/dnn_breast_cancer/dnn_breast_cancer.py [hunk 1]
```diff
@@ -40,7 +42,8 @@ def create_dnn():
 
 
 def get_estimator(epochs=50, batch_size=32, steps_per_epoch=None, validation_steps=None, model_dir=tempfile.mkdtemp()):
-    (x_train, y_train), (x_eval, y_eval) = tf.keras.datasets.boston_housing.load_data()
+    (X, y) = load_breast_cancer(True)
```
Comment (TortoiseHam) [inline apphub/tabular/dnn_breast_cancer/dnn_breast_cancer.py]: the original host of this dataset mentions that the included an ID column in their CSV. does the sklearn version get rid of that automatically?

Comment (TortoiseHam) [reply apphub/tabular/dnn_breast_cancer/dnn_breast_cancer.py]: nvm, looks like they handled it

File: apphub/tabular/dnn_breast_cancer/dnn_breast_cancer.ipynb [hunk 2]
```diff
@@ -0,0 +1,558 @@
...
+    "    ops=[ModelOp(inputs=\"x\", model=model, outputs=\"y_pred\"), BinaryCrossentropy(inputs=(\"y\",\"y_pred\"),outputs=\"loss\")])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 3: Configure training, create Estimator\n",
+    "\n",
+    "During the training loop, we want to: 1) measure lowest loss for data data 2) save the model with lowest valdiation loss. Trace class is used for anything related to training loop, we will need to import the ModelSaver trace."
```
Comment (TortoiseHam) [inline apphub/tabular/dnn_breast_cancer/dnn_breast_cancer.ipynb]: "loss for data data"
